app/assets/javascripts/IndividualResult.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+
+const [Number, String, Mixed, ObjectId] = [
+  mongoose.Schema.Types.Number,
+  mongoose.Schema.Types.String,
+  mongoose.Schema.Types.Mixed,
+  mongoose.Schema.Types.ObjectId,
+];
+
+const IndividualResultSchema = mongoose.Schema(
+  {
+    // Population Attributes
+    STRAT: Number,
+    IPP: Number,
+    DENOM: Number,
+    NUMER: Number,
+    NUMEX: Number,
+    DENEX: Number,
+    DENEXCEP: Number,
+    MSRPOPL: Number,
+    OBSERV: Number,
+    MSRPOPLEX: Number,
```

Comment:
Didn't we decide that there would be two versions of this? one for pop based and one for episode based? pop based would be boolean, episode based would be *the array of episodes*?

Also, doesn't OBSERV need to be an array of numbers?